asm/macros/battle_script.inc

```diff
@@ -1787,6 +1788,12 @@
 	.4byte \failInstr
 	.endm
 
+	.macro tidyupclear battler:req, clear:req, failInstr:req
+	various \battler, VARIOUS_TIDY_UP
+	.byte \clear
+	.4byte \failInstr
+	.endm
```

Comment:
I think it would make sense to separate this into two commands (idk what the names should be, but obviously they need to be different):
```
	.macro tidyupclear battler:req, failInstr:req
	various \battler, VARIOUS_TIDY_UP
	.byte FALSE
	.4byte \failInstr
	.endm

	.macro tidyupclear battler:req
	various \battler, VARIOUS_TIDY_UP
	.byte TRUE
	.4byte NULL
	.endm
```
Because `failInstr` isn't actually required in the `clear=TRUE` case.

But also we never actually use the `clear=FALSE` case in the code as-written. Why does it exist?

Reply:
Ahhh I think I see what `clear` does, if `TRUE` then it actually does the clearing, and if `FALSE` then it checks if there's anything that can be cleared and if there isn't it branches to `failInstr`?

I think it'd be fine to keep them both, and I suppose the names could be `tidyupclear` and `cantidyupclear` or something like that?